Justfile

```diff
@@ -56,12 +57,50 @@ audit:
 lint: && audit disallow-unused-cargo-deps
   shellcheck tests/regression
   shellcheck tests/stage-*
-  shellcheck ops/tier1-test
-  shellcheck ops/tier2-test
   shellcheck ops/productionize
+  shellcheck ops/ingest-all
   cd rust && cargo {{nightly_if_required}} fmt --all --check
   cd rust && cargo clippy --all-targets --all-features -- -D warnings
   [ "$(nixfmt < flake.nix)" == "$(cat flake.nix)" ]
 
-images:
-  ops/build-oci-image
+# Build OCI images.
+build-image:
+  #REV="$(git rev-parse --short=8 HEAD)"
```

Comment:
Did you mean these to be commented out?

Reply:
Yes, because the image build does not work. It was already commented out in these lines that were copied and pasted from a shell script that this PR deletes.